### ISSUE TITLE
Leave on hold and return.

### ISSUE THREAD

**Issue by gmolledaj** [flameshot-org/flameshot#1515]
Esperanto - English - Español
Mi uzas Flameshot por fari prezentojn interrete.
Estus interese havi klavon, kiu permesas ĉesigi la kaptan redaktadon por uzi la sistemon instruante ion alian aplikaĵon kaj poste povi reveni al nia loko, kun la redaktado farita ĝis nun por daŭrigi la redaktadon.

I use Flameshot to make presentations online.
It would be interesting to have a shortcut key that allows suspending the editing of the capture to use the system teaching something else and then be able to return to the point where we left it, with the editing done so far to continue with the editing.

Uso Flameshot para hacer presentaciones por internet.
Sería interesante una tecla que permita dejar en suspenso la edición de la captura para usar el sistema enseñando otra cosa y luego poder volver al punto donde lo dejamos, con la edición realizada hasta el momento para seguir editando.

**Comment by mmahmoudian** [comment]
@gmolledaj it has been [proposed](https://flameshotworkspace.slack.com/archives/C019W6EPYMN/p1614862767035800) to make it possible to store the screenshot and its annotations in a format like SVG so that users can move things around and modify annotations later in another software such as Inkscape or any other vector graphics editor. Do you think this proposal also covers your needs?

**Comment by gmolledaj** [comment]
Ne, ĉar mi uzas ĝin realtempe, se por la klarigo mi bezonas montri alian dokumenton kaj poste reiri al kie estis mia klarigo.
No, because I use it in real time, if for the explanation I need to show another document and then go back to where my explanation was.
No, porque lo uso en tiempo real, si para la explicación necesito enseñar otro documento y luego volver por donde iba.

Postskribo: Por la Fina Venko. Ha-ha-ha (terura voĉo).

**Comment by tom-vb** [comment]
yes please. Sometimes you take a screenshot and make annotations for a while, so would be nice been able to swap to another program and return again to work on the screenshot, or open the screenshot in a independent window like pin window but with editing tools enabled

**Comment by mmahmoudian** [comment]
I just want to connect this thread to #2215 as they are related.
